frontend/cypress/integration/featureFiles/wizard_request_routing.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Service Details Wizard: Request Routing
+
+  User navigates to the service details page and open the Kiali Wizard to generate a Request Routing scenario.
+
+  Background:
+    Given user is at administrator perspective
+
+  @wizard-request-routing
+  Scenario: Create a Request Routing scenario
+    When user opens the namespace "bookinfo" and "reviews" service details page
+    And user clicks in the "Request Routing" actions
+    And user sees the "Create Request Routing" wizard
```

Comment:
Should't there be pre-condition to remove existing routing first if exists?

Reply:
I assumed that the state was correct at the beginning of the test, and at the end of the test, the state was in the same position.

Otherwise, we'll need to add extra calls to update/delete config that with that assumption is shouldn't be necessary (and I guess it will add more rounds to our pipelines).

Reply:
> but one thing still is not clear for me, the step "And user types "100" traffic weight in the "reviews-v2" workload" on "Request Routing", is there wizard logic change?

Reply:
I mean, this component:
![image](https://user-images.githubusercontent.com/1662329/166957035-1e8f7cec-3129-47e4-9ecc-73232572c50b.png)

The test introduces the weight in the "input text" for simplicity

Reply:
Thanks for the review Hayk; I'm still working on a flaky test, it seems that I have a race condition.
I'll send some changes to see if it improves the testing in the CI, but It won't change the logic of the testing.